data/json/items/comestibles/med.json

```diff
@@ -545,6 +545,390 @@
+    "phase": "liquid",
+    "color": "green",
+    "addiction_potential": 5,
+    "flags": [ "NO_INGEST" ]
+  },
+  {
+    "id": "hi_q_distillate",
+    "type": "COMESTIBLE",
+    "comestible_type": "MED",
+    "name": { "str": "high quality distillate", "str_pl": "high quality distillate" },
+    "description": "Concentrated cannabis extract.  This style of extract goes through the most processing, eliminating most if not all harmful ingredients.  Normal cannabis contains around 20%THC, this distillate should be around 88%THC.  Can be smoked in a bowl, used in a dab pen, or used to make distillate cartridges.",
+    "looks_like": "honey_gold",
+    "weight": "1 g",
+    "volume": "1 ml",
+    "stack_size": 10,
+    "charges_per_use": 1,
+    "max_charges": 10,
+    "price": 1000,
+    "container": "jar_3l_glass",
+    "symbol": "!",
+    "color": "green",
+    "quench": -5,
+    "stim": -3,
+    "healthy": 0,
+    "fun": 15,
+    "phase": "liquid",
+    "addiction_potential": 5,
+    "flags": [ "NO_INGEST" ],
+    "use_action": {
+      "type": "consume_drug",
+      "activation_message": "You take a dab of some distillate.  Sooooo smooth...",
+      "effects": [
+        { "id": "weed_high", "duration": "180 m" },
+        { "id": "pkill1", "duration": "120 m" },
+        { "id": "pkill2", "duration": "5 m" },
+        { "id": "high", "duration": "60 m" },
+        { "id": "smoke", "duration": "1 m" }
+      ],
+      "stat_adjustments": { "hunger": 4, "thirst": 6 },
+      "fields_produced": { "fd_weedsmoke": 2 },
+      "charges_needed": { "fire": 1 },
+      "tools_needed": { "apparatus": -1 },
+      "moves": 6000
+    }
+  },
+  {
+    "id": "hi_q_distillate_cart",
+    "type": "COMESTIBLE",
+    "comestible_type": "MED",
+    "name": { "str": "high quality distillate cartridge" },
+    "description": "A small cartridge containing concentrated cannabis extract thats used with dab pens.  This style of extract goes through the most processing, eliminating most if not all harmful ingredients.  Normal cannabis contains around 20%THC, this distillate should be around 70%THC after reintroducing terpenes for flavoring.",
+    "looks_like": "canister_empty",
+    "weight": "1 g",
+    "volume": "1 ml",
+    "stack_size": 50,
+    "price": 5000,
+    "symbol": "!",
+    "color": "green",
+    "quench": -5,
+    "stim": -3,
+    "healthy": 0,
+    "fun": 15,
+    "addiction_potential": 5,
+    "flags": [ "NO_INGEST" ],
+    "use_action": {
+      "type": "consume_drug",
+      "activation_message": "You hit your dab pen.  Sooooo smooth...",
+      "effects": [
+        { "id": "weed_high", "duration": "90 m" },
+        { "id": "pkill1", "duration": "90 m" },
+        { "id": "pkill2", "duration": "5 m" },
+        { "id": "high", "duration": "45 m" },
+        { "id": "smoke", "duration": "1 m" }
+      ],
+      "stat_adjustments": { "hunger": 3, "thirst": 5 },
+      "fields_produced": { "fd_weedsmoke": 2 },
+      "need_charges": 1,
```

Comment:
```suggestion
      "charges_needed": { "fire": 1 },
```

Reply:
This particular comestible uses the Dab Pen (On) as the tool which runs on battery power so a fire source shouldn't be required.

Reply:
^